<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgDecideKick</class>
 <widget class="QDialog" name="DlgDecideKick">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>保留组需要保留哪个译文?</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>原文</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edt_src">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lab_retain">
     <property name="text">
      <string>(译文)保留组, 将保留条目 - %1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edt_trs_retain"/>
   </item>
   <item>
    <widget class="QLabel" name="lab_kick">
     <property name="text">
      <string>(译文)剔除组, 将删除条目 - %1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edt_trs_kick"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chb_following">
       <property name="text">
        <string>之后也这么做</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_retain">
       <property name="text">
        <string>保留组的</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_kick">
       <property name="text">
        <string>剔除组的</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
